rwengine/src/objects/VehicleObject.cpp

```diff
@@ -29,6 +29,16 @@ VehicleObject::VehicleObject(GameWorld* engine, const glm::vec3& pos, const glm:
 {
 	collision = new CollisionInstance;
 	if( collision->createPhysicsBody(this, data->modelName, nullptr, &info->handling) ) {
+
+		// calculate the intertia from the handling information
+		auto& dim = info->handling.dimensions;
+		auto mass = info->handling.mass;
+		btVector3 handlingInertia(
+		    1.f / ((mass / 12) * (dim.y * dim.y + dim.z * dim.z)),
```

Comment:
https://en.wikipedia.org/wiki/List_of_moments_of_inertia your guess is as good as mine :8ball: 

I'll leave a note.